Clicker/SectionControllers/PollOptionsCellSectionControllers/AskQuestionSectionController.swift

```diff
@@ -38,6 +40,7 @@ class AskQuestionSectionController: ListSectionController {
     override func cellForItem(at index: Int) -> UICollectionViewCell {
         let cell = collectionContext?.dequeueReusableCell(of: AskQuestionCell.self, for: self, at: index) as! AskQuestionCell
         cell.configure(with: self, askQuestionModel: askQuestionModel)
+        cell.delegate = self
```

Comment:
can't we set the delegate from the `configure` function since we already pass `self` in through it